angular/src/lib/editable-textfield/editable-textfield.component.ts

```diff
@@ -0,0 +1,194 @@
+import { Component, ElementRef, OnInit, Input, Output, EventEmitter, Optional, Self, ViewChild} from '@angular/core';
+
+import { ControlValueAccessor, NgControl } from '@angular/forms';
+
+const cb = () => {};
+@Component({
+  selector: 'cui-editable-textfield',
+  template: `
+    <span
+      class="cui-editable-textfield"
+      [ngClass]="wrapperClasses"
+    >
```

Comment:
The span can be removed and bind class to host.

Reply:
removed and updated to new pattern